packages/components/src/icon/index.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Jupyter Development Team.
+// Copyright (c) Microsoft Corporation.
+// Distributed under the terms of the Modified BSD License.
+
+import {
+  FASTElement,
+  customElement,
+  attr,
+  html
+} from '@microsoft/fast-element';
+
+const template = html<Icon>`
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    innerHTML="${x => x.getSvg()}"
+  ></svg>
```

Comment:
You should not set the SVG content as the icon will register a full SVG This means that should look like:

```suggestion
  ${x => x.getSvg()}
```

you can see the issue in the storybook (execute `yarn start` in `packages/components`):

![image](https://github.com/user-attachments/assets/1ed67f31-069f-412d-8468-0aa2d9edde3a)

> FYI `innerHTML` is a property of an HTML element not an attribute. Therefore you can not set it as you did here.